ui/pages/settings/info-tab/info-tab.component.js

```diff
@@ -53,6 +57,13 @@ export default class InfoTab extends PureComponent {
   componentDidMount() {
     const { t } = this.context;
     handleSettingsRefs(t, t('about'), this.settingsRefs);
+    if (this.props.remoteFeatureFlags.length > 0) {
```

Comment:
Will be removed after 1st feature flag is implemented in production

Reply:
Instead of (this.props.remoteFeatureFlags.length > 0, can we use an actual feature flag. So that this becomes something like

```
if (this.props.remoteFeatureFlags[TEST_FLAG] > 0) {
    console.log(
```

So we can fully demonstrate how teams would use the feature flag system?

Reply:
Good idea, added 5204e9c4ff436e00a8f512319bcf0c23b88d6393